### PR TITLE
nextgenhealthcare#5629 add reverse proxy configuration

### PR DESCRIPTION
**Pull request by gbenosman**

### CODE REVIEW

**Comment by jonbartels** [comment]
Linking the issue https://github.com/nextgenhealthcare/connect/issues/5629

File: server/src/com/mirth/connect/server/servlets/WebStartServlet.java [hunk 1]
```diff
@@ -182,9 +182,16 @@ protected Document getAdministratorJnlp(HttpServletRequest request) throws Excep
         String serverHostname = request.getServerName();
         int serverPort = request.getServerPort();
         String contextPath = request.getContextPath();
-        String codebase = scheme + "://" + serverHostname + ":" + serverPort + contextPath;
 
         PropertiesConfiguration mirthProperties = getMirthProperties();
+        
+        String codebase = null;
+        if (StringUtils.isNotBlank(mirthProperties.getString("codebase.webstart.url"))) {
```
Comment (jonbartels) [inline server/src/com/mirth/connect/server/servlets/WebStartServlet.java]: PropertiesConfiguration supports[ a getString with a defaultValue argument ](https://commons.apache.org/proper/commons-configuration/apidocs/org/apache/commons/configuration2/AbstractConfiguration.html#getString(java.lang.String,java.lang.String)) 

Would using this getString method be better than an if/else condition? it is logically the same.

Comment (gbenosman) [reply server/src/com/mirth/connect/server/servlets/WebStartServlet.java]: Yes no problem !

Comment (gbenosman) [reply server/src/com/mirth/connect/server/servlets/WebStartServlet.java]: I pushed the modification, I did the test and it works

**Review by jonbartels — APPROVED** [review]

**Comment by gbenosman** [comment]
Hello
@pladesma , @jonbartels Could you please merge the pull request please.
I don't have the rights for, I didn't see that the MR was not validate before sorry.
Thank you

**Review by joaryche — APPROVED** [review]

**Pull request merged**